package.json

```diff
@@ -47,7 +47,7 @@
     "swagger-ui-express": "^4.3.0"
   },
   "engines": {
-    "node": "16"
+    "node": "18"
```

Comment:
According to https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines, 
changing this field is a breaking change.
Check, but maybe the value should be `>=16`?